src/components/Banner.tsx

```diff
@@ -65,15 +83,17 @@ function Banner({text, onClose, onPress, containerStyles, textStyles, shouldRend
                         ]}
                     >
                         <View style={[styles.flexRow, styles.flexGrow1, styles.mw100, styles.alignItemsCenter]}>
-                            {shouldShowIcon && (
+                            {shouldShowIcon && icon && (
                                 <View style={[styles.mr3]}>
                                     <Icon
-                                        src={Expensicons.Exclamation}
+                                        src={icon}
                                         fill={StyleUtils.getIconFillColor(getButtonState(shouldHighlight))}
                                     />
                                 </View>
                             )}
-                            {shouldRenderHTML ? (
+                            {content && content}
```

Comment:
NAB: Should we use a Boolean here? I feel like it looks a bit more readable that way 
```suggestion
                            {!!content && content}
```